sandbox/cgroup/scanning.go

```diff
@@ -67,33 +68,25 @@ func securityTagFromCgroupPath(path string) naming.SecurityTag {
 	return nil
 }
 
-// PidsOfSnap returns the association of security tags to PIDs.
-//
-// NOTE: This function returns a reliable result only if the refresh-app-awareness
-// feature was enabled before all processes related to the given snap were started.
-// If the feature wasn't always enabled then only service process are correctly
-// accounted for.
-//
-// The return value is a snapshot of the pids for a given snap, grouped by
-// security tag. The result may be immediately stale as processes fork and
-// exit.
-//
-// Importantly, if the per-snap lock is held while computing the set, then the
-// following guarantee is true: if a security tag is not among the results then
-// no such tag can come into existence while the lock is held.
+type InstancePathsFlags uint32
+
+const (
+	InstancePathsFlagsOnlyPaths InstancePathsFlags = 1 << iota
```

Comment:
we tend to use *Options structs with bool fields for this kind of things

Reply:
Done.